platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/TrackingSettings.java

```diff
@@ -306,7 +306,7 @@ public boolean isDismissTrackingModesForCameraPositionChange() {
    * <p>
    * </p>
    *
-   * @param willAllowToDismiss True will allow animated camera changes dismiss a trackig mode
+   * @param willAllowToDismiss True will allow animated camera changes dismiss a tracking mode
    */
   public void setDismissTrackingModeForCameraPositionChange(boolean willAllowToDismiss) {
```

Comment:
both setter and getter for `isResetTrackingWithCameraPositionChange` can be removed completely